install/mobile-network-performance/install.yml

```diff
@@ -0,0 +1,17 @@
+id: mobile-network-performance-install
```

Comment:
Sorry didn't catch this on first review, but we're actually deprecating the use of install plans in favor of data sources. 
Here's a guide on the conversion: 
https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#install-plans-deprecated